src/log/log.c

```diff
@@ -1916,7 +1916,7 @@ __log_has_hole(WT_SESSION_IMPL *session, WT_FH *fh, wt_off_t log_size,
 			 * present.  This record may also be partially written.
```

Comment:
I think this comment here needs more based on my new understanding of this code today. What we're NOT doing is determining whether the rest of the file is corrupted. What we ARE doing is looking at the first non-zero bit that made us decide we have a hole and determining if that first non-zero part is a valid log record header (i.e. it is a hole) or if the first non-zero part is corruption/flipped bits, etc. I suggest:
/*
 * A presumed log record begins here where the buffer becomes non-zero. If we have
 * enough of a log record present in the buffer, we either have a valid header or corruption.
 * Verify the header of this record to determine whether it is just a hole or corruption.
 */